src/xradio/vis/_vis_utils/_ms/partition_queries.py

```diff
@@ -14,6 +14,137 @@
 from .subtables import subt_rename_ids
 
 
+def enumerated_product(*args):
+    yield from zip(
+        itertools.product(*(range(len(x)) for x in args)), itertools.product(*args)
+    )
+
+
+def create_partitions(in_file: str, partition_scheme: Union[str, list], vla_otf=False):
+    """Create a list of dictionaries with the partition information.
+
+    Args:
```

Comment:
This looks more like google docstring style than numpy style. If this was not intentional I'll reformate it when reorganizing this file.